content/docs/plugins.md

```diff
@@ -117,7 +117,7 @@ Description:
 	This function should return the name which should be displayed in the LanguageMenu.   
 	If more than one lexer is contained in a plugin, this function will be called accordingly often  
 	and the respective lexer is queried via index.   
-	Name is a pointer to the memory area allocated by Npp.   
+	Name is a pointer to the memory area allocated by Npp. 8bit encoded name is expected.  
```

Comment:
The phrase "8bit encoded name is expected" is awkward.
I might suggest "This name should be encoded as an 8bit string using the current Windows codepage."